.github/workflows/master.yml

```diff
@@ -0,0 +1,150 @@
+name: master
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches: 
+      - master
+
+jobs:
+  python35:
+    runs-on: ubuntu-latest
+    env:
+      SPARK_VERSION: 2.3.4
+      PANDAS_VERSION: 0.23.4
+      PYARROW_VERSION: 0.10.0
+      PYTHON_EXECUTABLE: xvfb-run python
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    # Setup Python 3.5 via `apt-get install` since the dafault Python3.5 from `actions/setup-python`
+    # seems to have some problem with Tkinter, so we should manually install the python3.5-tk.
+    # for this, we should use the Python we manually installed, not the default one from `actions/setup-python`
+    - name: Setup Python 3.5
+      run: |
+        sudo add-apt-repository ppa:deadsnakes/ppa
+        sudo apt-get install tk-dev python3.5-tk python3.5
+        sudo rm /usr/bin/python
+        sudo ln -s /usr/bin/python3.5 /usr/bin/python
+        # Add the path which packages are actually installed to `PATH`
```

Comment:
Can you clarify this `echo "::add-path::/home/runner/.local/bin"` was required for pip installed executables to be in the path?

Reply:
okay i'll fix it to work dynamically. thanks 👍